app/src/main/java/com/hedvig/app/ui/compose/composables/buttons/PrimaryButtons.kt

```diff
@@ -11,15 +11,33 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.dimensionResource
 import androidx.compose.ui.tooling.preview.Preview
-import com.airbnb.android.showkase.annotation.ShowkaseComposable
 import com.hedvig.app.R
 import com.hedvig.app.ui.compose.theme.HedvigTheme
 
 @Composable
-fun LargeContainedButton(
+fun PrimaryTextButton(
```

Comment:
I think we should use the terminology from the design system - that is:

- (Large) Contained Button
- (Large) Outlined Button
- Text Button

Reply:
Very reasonable.